### PR TITLE
Separated Lint tests

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,10 @@ virtualenv:
   system_site_packages: true
 
 env:
-  - VERSION="8.0" ODOO_REPO="odoo/odoo"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB"
-  - VERSION="8.0" UNIT_TEST="1"
+  - VERSION="8.0" LINT_CHECK="1"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="8.0" UNIT_TEST="1" LINT_CHECK="0"
```
Comment (dreispt) [inline .travis.yml:12]: Can't `LINT_CHECK="0"` be removed from these lines?
Simple is better.

Comment (yvaucher) [inline .travis.yml:12]: For now we must specify LINT_CHECK I think it makes sense for retro compatibility.

**Comment by yvaucher** [comment]
Few pylint errors

File: .travis.yml (hunk 1 above)
Comment (pedrobaeza) [inline .travis.yml:12]: Yeah, I put that it has to be explicitely marked to not make the pylint checks, to keep compatibility with branches not converted to this new style, as the current policy states.

**Comment by pedrobaeza** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (dreispt) [inline .travis.yml:12]: Yes, I confirm that.
:+1: then

**Comment by dreispt** [comment]
Thanks for the cleanup.
I'm merging.

**Pull request merged**